vshard/service_info.lua

```diff
@@ -44,6 +44,7 @@ local function service_set_status_ok(service)
     service_set_status(service, 'ok')
     service_next_iter(service)
     service.error = ''
+    service.last_info = ''
```

Comment:
This seems to be unused. What is it for? Can we still keep the info messages? They might tell not less than the error logs. If they are spamming, then the nop-messages (like when 0 is discovered, 0 is recovered, etc) could be dropped. And the others somehow aggregated, perhaps.